src/sentry/web/frontend/accounts.py

```diff
@@ -290,7 +276,6 @@ def appearance_settings(request):
 @csrf_protect
 @never_cache
 @login_required
-@sudo_required
 @transaction.atomic
 def notification_settings(request):
```

Comment:
Notification settings should require sudo since you can send to arbitrary mail addresses there.